res/xml/bengali_provat.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<keyboard>
+	<row>
+		<key 				key0="দ"	key1="ধ"	key2="১"	key3="!"				key5="esc"	/>
+		<key 				key0="ূ"		key1="ঊ"	key2="২"	key3="‌‌‌‌@"							/>
+		<key 				key0="ী"		key1="ঈ"	key2="৩"	key3="‌‌‌‌#"							/>
```

Comment:
These can be properly escaped:
```suggestion
		<key 				key0="ূ"		key1="ঊ"	key2="২"	key3="\@"							/>
		<key 				key0="ী"		key1="ঈ"	key2="৩"	key3="\#"							/>
```

Reply:
Thanks. On my editor indentation was okat, but don't know what happen after commit.